src/main/java/bank/account/AccountRepository.java

```diff
@@ -0,0 +1,15 @@
+package bank.account;
+
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface AccountRepository extends JpaRepository<Account, Long> {
+    Optional<Account> findByUid(String uid);
+
+    Optional<Account> findByUidAndPersonId(String uid, Long person_id);
```

Comment:
1. не впевнений що воно працює взагалі 
Має бути ось так: 
```suggestion
    Optional<Account> findByUidAndPerson(String uid, Person person);
```
2. В теорії uid має бути унікальним, тож майже немає сенсу мати методи findByUid та findByUidAndPerson 
Другий лише перевірить, що аккаут належить конкретній персоні.